MANIFEST.SKIP

```diff
@@ -0,0 +1,72 @@
+
+#!start included /home/jberger/perl5/perlbrew/perls/perl-5.22.1/lib/5.22.1/ExtUtils/MANIFEST.SKIP
```

Comment:
should this be generalized?

Reply:
That's an auto-generated comment by helper tools. The path in there can be ignored, it's informational only.

Reply:
You've discovered some of the really dumb things about the MANIFEST.SKIP system. The original skip file is generated and it imprints the file location of the original that it included as a comment. Yes that's just a comment you could remove it but meh. The .bak file happens when you generate it twice and it tries very hard not to overwrite, at least once.

Reply:
None of it should matter and for internal tooling I'd just leave it alone.